OSDOCS-12623-update: Added extra parameters to the Ansible migration … (#95989)
openshift/openshift-docs: modules/nw-ovn-kubernetes-ansible-migration.adoc

+
`migration_interface_name`:: If you use an `NodeNetworkConfigurationPolicy` (NNCP) resource on a primary interface, specify the interface name in the `migration-playbook.yml` file so that the NNCP resource gets deleted on the primary interface during the migration process.
`migration_disable_auto_migration`:: Disables the auto-migration of OpenShift SDN CNI plug-in features to the OVN-Kubernetes plugin. If you disable auto-migration of features, you must also set the `migration_egress_ip`, `migration_egress_firewall`, and `migration_multicast` parameters to `false`. If you need to enable auto-migration of features, set the parameter to `false`. 

> `migration_routing_via_host`:: A value of `true` sets local gateway mode and a value of `false` sets shared gateway mode for nodes in your cluster. The default value is `false`. In local gateway mode, traffic is routed through the host network stack. In shared gateway mode, traffic is not routed through the host network stack.

Reviewer: ```suggestion
`migration_routing_via_host`:: Set to `true` to configure local gateway mode or `false` to configure shared gateway mode for nodes in your cluster. The default value is `false`. In local gateway mode, traffic is routed through the host network stack. In shared gateway mode, traffic is not routed through the host network stack.
```
